sympy/physics/optics/polarization.py

```diff
@@ -0,0 +1,609 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+r"""
+The module implements:
+
+- Jones vectors.
+
+- Stokes vectors.
+
+- Jones matrices.
+
+- Mueller matrices.
+
+Please see the description of the individual functions for further
+details and examples.
```

Comment:
The new documentation guide
https://docs.sympy.org/dev/documentation-style-guide.html
says
```
The module level docstring should discuss the purpose and scope of the module, and give a high-level example of how to use the functions or classes in the module
```
I expected there to be more guidance there about module level docstrings but when I looked this is what I found.

Reply:
I just added further description of the module and examples.